src/Event/EventRegistrationCommandHandler.php

```diff
@@ -40,12 +40,14 @@ public function handle(EventRegistrationCommand $command): void
 
         $this->manager->create($registration = $this->factory->createFromCommand($command));
 
-        $this->mailer->sendMessage(EventRegistrationConfirmationMessage::createFromRegistration(
-            $registration,
-            $eventLink = $this->generateUrl('app_event_show', [
-                'slug' => $command->getEvent()->getSlug(),
-            ])
-        ));
+        if ($sendMail) {
```

Comment:
I think this was done for another handler, but it would be better to trigger an event here holding that boolean et let a listener send the email.